code/modules/admin/admin_verbs.dm

```diff
@@ -782,12 +782,12 @@ var/global/list/admin_verbs_mod = list(
 		M.skin_tone =  -M.skin_tone + 35
 
 	// hair
-	var/decl/sprite_accessory/new_hstyle = input(usr, "Select a hair style", "Grooming") as null|anything in decls_repository.get_decls_of_subtype(/decl/sprite_accessory/hair)
+	var/decl/sprite_accessory/new_hstyle = input(usr, "Select a hair style", "Grooming") as null|anything in decls_repository.get_decl_paths_of_subtype(/decl/sprite_accessory/hair)
 	if(new_hstyle)
 		M.h_style = new_hstyle.type
```

Comment:
```suggestion
		M.h_style = new_hstyle
```
It's returning a path now, but then still trying to get the type of the path, which is invalid.